build_ext/lint.py

```diff
@@ -359,6 +360,110 @@ def err(self, node, msg=None):
         return ret
 
 
+def detect_overindent(logical_line, tokens, indent_level, hang_closing, indent_char, noqa, verbose):
```

Comment:
Well, if you cherry-pick it, then it becomes a new commit; whichever change change is merged later thus generally becomes a no-op. I guess at this point if it's really painful to back-out though, let's not bother (and IIRC, you have this check blacklisted for now). We can simply make the scope of turning it on a subsequent PR.